pubsub/pulsar/pulsar_test.go

```diff
@@ -17,6 +18,21 @@ func TestParsePulsarMetadata(t *testing.T) {
 	assert.Equal(t, false, meta.EnableTLS)
 }
 
+func TestParsePublishMetadata(t *testing.T) {
+	m := &pubsub.PublishRequest{}
+	m.Metadata = map[string]string{
+		"deliverAt":    "2021-08-31T11:45:02Z",
+		"deliverAfter": "60s",
+	}
+	msg, err := parsePublishMetadata(m)
+	assert.Nil(t, err)
+
+	val, _ := time.ParseDuration("60s")
```

Comment:
Could we use things like `60 * time.minutes`?

Reply:
Yes, 60m

Reply:
seconds, sorry.